### PR TITLE
UCP/WIREUP: Remove signaling proxy ep

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

File: src/ucp/wireup/wireup.c [hunk 1]
```diff
@@ -726,40 +726,37 @@ ucp_wireup_connect_lane_to_iface(ucp_ep_h ep, ucp_lane_index_t lane,
...
-            status = uct_ep_create(&uct_ep_params, &uct_ep);
-            if (status != UCS_OK) {
-                /* coverity[leaked_storage] */
-                return status;
-            }
-
-            ucp_wireup_assign_lane(ep, lane, uct_ep, "");
-        }
+    if ((ep->uct_eps[lane] != NULL) && !ucp_wireup_ep_test(ep->uct_eps[lane])) {
+        /* Lane already exists */
```
Comment (dmitrygx) [inline src/ucp/wireup/wireup.c:736]: minor: lowercase

File: src/ucp/wireup/signaling_ep.c [hunk 2]
```diff
@@ -1,208 +0,0 @@
...
-    status = uct_ep_tag_rndv_request(proxy_ep->uct_ep, tag, header,
-                                     header_length,
-                                     flags | UCT_SEND_FLAG_SIGNALED);
-    if (status == UCS_OK) {
-        ucp_proxy_ep_replace(proxy_ep);
-    }
-    return status;
-}
-
-ucs_status_t ucp_signaling_ep_create(ucp_ep_h ucp_ep, uct_ep_h uct_ep,
```
Comment (dmitrygx) [inline src/ucp/wireup/signaling_ep.c:192]: remove the function declaration from `wireup.h` file

Comment (yosefe) [reply src/ucp/wireup/signaling_ep.c:192]: fixed

File: src/ucp/wireup/wireup.c (hunk 1 above)
Comment (yosefe) [reply src/ucp/wireup/wireup.c:736]: actually, comments are supposed to start with uppercase

Comment (dmitrygx) [reply src/ucp/wireup/wireup.c:736]: I see

**Review by dmitrygx — APPROVED** [review]

**Review by brminich — APPROVED** [review]
will you remove the rest in the following PRs?

**Comment by yosefe** [comment]
> will you remove the rest in the following PRs?

@brminich i thought i removed everything here.. what did i miss?

**Comment by brminich** [comment]
> > will you remove the rest in the following PRs?
> 
> @brminich i thought i removed everything here.. what did i miss?

my mistake, had a wrong impression there is bunch of ucp_proxy funcs relevant for signalling eps only

**Pull request merged**